personio-personnel-data-api-oa3.yaml

```diff
@@ -3469,6 +3634,121 @@ components:
                     example: TimeOffPeriod
                   attributes:
                     $ref: "#/components/schemas/Absence"
+    HourlyAbsencePeriodsResponse:
+      title: List All Hourly Absence Periods response
+      type: object
+      allOf:
+        - $ref: "#/components/schemas/Response"
+        - type: object
+          properties:
+            metadata:
+              type: object
+              required:
+                - total_elements
+                - current_page
+                - total_pages
+              properties:
+                total_elements:
+                  type: integer
+                  description: The total number of items matching the query filters.
+                total_pages:
+                  type: integer
+                  description: The total number of pages distributing sets of items matching the query filters.
+                current_page:
+                  type: integer
+                  description: Current page number containing the returned set of items matching with the query filters. The page number range is from 0 to total_pages - 1
+            offset:
+              type: integer
+            limit:
+              type: integer
+            data:
+              type: array
+              items:
+                type: object
+                required:
+                  - type
+                  - attributes
+                properties:
+                  type:
+                    example: AbsencePeriod
+                  attributes:
+                    $ref: "#/components/schemas/HourlyAbsence"
+    HourlyAbsencePeriodResponseAttributes:
+      type: object
+      required:
+        - id
+        - employee
+        - absence_type_id
+        - start
+        - half_day_start
+        - half_day_end
+        - origin
+        - status
+        - certificate
+        - created_by
+        - effective_duration
+        - created_at
+        - updated_at
+        - breakdowns
+      properties:
+        id:
+          type: string
+          example: 9bba303f-0fbc-4514-9958-0befa21923fb
+        measurement_unit:
```

Comment:
Can be an enum with only value `minutes` for this version? Should be minutes, not hours, since we want to stay with integer numbers, right?